packages/atomic/cypress/e2e/result-list/result-components/result-badge.cypress.ts

```diff
@@ -42,9 +42,6 @@ describe('Result Badge Component', () => {
           .init();
       });
 
-      CommonAssertions.assertRemovesComponent(() =>
```

Comment:
Why are the following being removed? Is it part of the "To Test" of the PR?

Reply:
These assertions were specifically added by @btaillon a while back to confirm that this method was giving us false positives 🙂 

You can see where he added them in the Jira description:
https://coveord.atlassian.net/browse/KIT-2497